install/installer/pkg/components/blobserve/configmap.go

```diff
@@ -70,6 +58,10 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 						Search:      "open-vsx.org",
 						Replacement: openVSXProxyUrl,
 						Path:        "/ide/out/vs/workbench/workbench.web.main.js",
+					}, {
+						Search:      "ide.gitpod.io/code/markeplace.json",
+						Replacement: fmt.Sprintf("ide.%s/code/marketplace.json", ctx.Config.Domain),
```

Comment:
@iQQBot `ide.%s` is it correct? In Self-Hosted we proxy via proxy on `ide` subpath?

Reply:
Can we somehow test it with self-hosted?